src/ImageSharp/Processing/Transforms/Resamplers/BicubicResampler.cs

```diff
@@ -5,7 +5,7 @@ namespace SixLabors.ImageSharp.Processing
 {
     /// <summary>
     /// The function implements the bicubic kernel algorithm W(x) as described on
-    /// <see href="https://en.wikipedia.org/wiki/Bicubic_interpolation#Bicubic_convolution_algorithm">Wikipedia</see>
+    /// <seealso href="https://en.wikipedia.org/wiki/Bicubic_interpolation#Bicubic_convolution_algorithm">Wikipedia</seealso>
```

Comment:
Nit. IMO this should be `<see/>`.

Reply:
I wave to change all the links on the project because it wouldn’t compile because errors on the <see> <seealso> ware invalid because of invalid “http:” and “https:” on them.
I used a lot of “search and replace” to solve this problem on the entire project.
This problem disappear after I removed GhostDoc.
I have problems with some assignments like: MyType<TPixel> x = new MyType<TPixel>();
VS marked as an error and suggested ignore the this error or chage the code to: var x = new MyType<TPixel>();
I used the “var” suggestion. The version that I’m using now (VS2016 v15.3.5) reports as a warning